site/source/tutorials/006_deploying_to_production/index.md

```diff
@@ -63,7 +63,7 @@ There are four groups of command line arguments that are available when using th
 
 The first group is `--mode` that accepts `dist`, `dev`, `unit` and `functional` depending on the bundle that is needed, the argument defaults to `dist`.
 
-The build command supports two modes of `--watch`, `file` which writes built files to the relevant directory in `output` and `memory` that uses webpack dev server (and must be run with the `--serve` argument).
+The build command `--watch` which builds files to the relevant directory in `output` (and must be run with the `--serve` argument).
```

Comment:
The `--watch` command doesn't have to be run with the `--serve` argument. If the CLI doesn't serve it then the browser refresh won't work but the watch still will

Reply:
Cool, alright I'll take that bit out